### ISSUE TITLE
Media queries issues

### ISSUE THREAD

**Issue by andreamangano** [Objectway/super-gigi#3]
The following kind of media query doesn't work:

```
  @include media-query(breakpoint1, breakpoint2) {
    //- code here
  }
```
I use the 1.9.0 version.

@MakhBeth Do you know more about the issue?

**Comment by MakhBeth** [comment]
Thanks, it's a problem with Gulp Sass 2.3.1
I have to investigate about it...

**Comment by MakhBeth** [comment]
Ok, it was a variables scoping issue...
Resolved in 1.11.2 version. 
Thanks for the help
